django/db/models/functions/__init__.py

```diff
@@ -20,8 +24,11 @@
     'ExtractYear', 'Now', 'Trunc', 'TruncDate', 'TruncDay', 'TruncHour',
     'TruncMinute', 'TruncMonth', 'TruncQuarter', 'TruncSecond', 'TruncTime',
     'TruncYear',
+    # numeric
```

Comment:
`Abs` is already proposed in #9577.

Reply:
That was a mistake.  I'll remove it in my next commit.